### ISSUE TITLE
use SWC configuration problems and solutions material

### ISSUE THREAD

**Issue by wikfeldt** [coderefinery/installation#35]
Many of the issues we encountered in Trondheim seem to have solutions 
[in this software carpentry page](https://github.com/swcarpentry/workshop-template/wiki/Configuration-Problems-and-Solutions)

**Comment by Vathasav** [comment]
I have tested the following steps in windows laptop. We can show them at the beginning of the workshop to check if python and Anaconda scrips - sphinx-quickstart, jupyter, etc are available in git bash.

- open windows command prompt
- type "where python” to check where python is installed
   - If you use Anaconda, usually it is : “C:\Users\<username>\Anaconda2\python.exe"

**Add python and anaconda scripts to your path**

```shell
SETX PATH “%PATH%;C:\Users\<username>\Anaconda2\Scripts;C:\Users\<username>\Anaconda2”
```
Close the windows command prompt and open a new one.

Check python --version

If you open git bash, you will have python and anaconda scripts available to git bash.

**Comment by bast** [comment]
Awesome doc! Can you please place submit it so that we have it on the website? No problem if it's too much work at this moment, in this case please signal this and somebody else will do it but this will help a lot. Thank you!

**Comment by bast** [comment]
I believe this is solved in #36?

**Comment by wikfeldt** [comment]
indeed solved in #36 
closing issue
